src/test/scala/de/zalando/beard/filter/FilterSpec.scala

```diff
@@ -0,0 +1,36 @@
+package de.zalando.beard.filter
+
+import org.scalatest.{FunSpec, Matchers}
+
+/**
+  * @author boopathi
+  */
+class FilterSpec extends FunSpec with Matchers {
+  def getFilter(name: String) =
+    DefaultFilterResolver().resolve(name, Set.empty) match {
+      case Some(filter) => filter
+      case None => throw FilterNotFound(name)
+    }
+  describe("filters") {
+    it("capitalize") {
+      val filter = getFilter("capitalize")
+      filter.apply("capitalize") should be("Capitalize")
+      filter.apply("cAPITALIZE") should be("CAPITALIZE")
+    }
+    it("uppercase") {
```

Comment:
the convention for BDD is that the it clauses should be readable... `it ("should upppercase")` in this case